src/http-operation.ts

```diff
@@ -0,0 +1,199 @@
+import { INode, INodeExample } from './graph';
+import { ISchema } from './schemas';
+import { IServer } from './servers';
+
+export interface IHttpOperation extends INode {
+  method: string;
+  path: string;
+  responses: IHttpOpResponse[];
+  servers?: IServer[];
+  request?: IHttpOpRequestParams;
+  security?: HttpOpSecurityScheme[];
+  deprecated?: boolean;
+}
+
+// Inspired by: https://github.com/OAI/OpenAPI-Specification/blob/master/versions/3.0.0.md#parameterObject
+export interface IHttpOpParam {
```

Comment:
Yeahh, I see your point. Going `Operation` would mean that things get preetttyy long though: `IHttpOperationRequestParams`. I spelled out the main interface (`IHttpOperation` instead of `IHttpOp`) with the hope that perhaps we could abbreviate for the more specific internals after establishing the long name. Regarding `Options` confusion, I normally abbreviate options to `Opts`, but I can see the potential confusion. O versus 0 I have no answer for.

Let's get votes from @XVincentX, @StefanDywersant, and @pytlesk4 (need all three to make the vote an odd number!) and just go with the winning one :).

Reply:
Operation - `IHttpOperationParam`

Reply:
I do not mind as long are consistent. So far we haven't really been, though.

Reply:
@marbemac I vote for `Operation` but I don't want to be a blocker here if there is no consensus, so please go with what your intuition tells you. 

"Follow your heart!" :)